src/main/java/net/sf/jabref/logic/exporter/ExportFormat.java

```diff
@@ -310,7 +324,7 @@ public void performExport(final BibDatabaseContext databaseContext, final String
             }
 
             // Clear custom name formatters:
-            Globals.prefs.customExportNameFormatters = null;
+            layoutPreferences.getCustomExportNameFormatters().clear();
```

Comment:
This list is local to the preferences object, so it should work. Earlier on it was a static variable in JabRefPreferences, but now it is created with the LayoutFormatPreferences object. I still think it should be cleared since the same LayoutFormatPreferences object may be passed to several classes at different times.